polynote-frontend/polynote/ui/component/about.ts

```diff
@@ -196,6 +191,60 @@ export class About extends FullScreenModal {
         return el;
     }
 
+    stateInspector() {
```

Comment:
This is pretty useful for debugging - a new tab in About that shows the State of the entire UI.